### PR TITLE
build: plugin 6.3.5, gradle 8.0.2, standardized project names, jakarta validation

### PR DESCRIPTION
**Pull request by wetted**

### CODE REVIEW

**Review by sdelamo — APPROVED** [review]

File: object-storage-gcp/src/test/groovy/io/micronaut/objectstorage/googlecloud/GoogleCloudStorageFakeGcsServerSpec.groovy [hunk 1]
```diff
@@ -33,11 +28,11 @@ class GoogleCloudStorageFakeGcsServerSpec extends AbstractGoogleCloudStorageSpec
 
     @Shared
     @AutoCleanup
-    static GenericContainer fakeGcs = new FixedHostPortGenericContainer('fsouza/fake-gcs-server:1.40.1')
-            .withFixedExposedPort(port, 4443)
+    static final GenericContainer<?> fakeGcs = new GenericContainer<>("fsouza/fake-gcs-server")
```
Comment (alvarosanchez) [inline object-storage-gcp/src/test/groovy/io/micronaut/objectstorage/googlecloud/GoogleCloudStorageFakeGcsServerSpec.groovy:31]: Why was this changed? Is `FixedHostPortGenericContainer` no longer working?

In any case, if this new version works, I'm fine with it. But then, the `@Shared` variable `port` needs to be removed.

Comment (wetted) [reply object-storage-gcp/src/test/groovy/io/micronaut/objectstorage/googlecloud/GoogleCloudStorageFakeGcsServerSpec.groovy:27]: That's correct. It was failing and with the change I made it worked again. I have removed the `port` variable.

**Review by alvarosanchez — APPROVED** [review]
